### PR TITLE
Adds a glossary entry for 'codebase'

### PR DESCRIPTION
**Pull request by bredamc**
Addresses issue #319

### CODE REVIEW

**Review by bburt-rh — DISMISSED** [review]

**Comment by bburt-rh** [comment]
lgtm

**Review by mportman12 — APPROVED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc [hunk 1]
```diff
@@ -479,6 +479,17 @@
 
 *See also*:
 
+[discrete]
+[[codebase]]
+==== image:images/yes.png[yes] codebase (noun)
+*Description*: A _codebase_ is a complete collection of source code for a software component, application, or system. Write as shown: two words.
```
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:485]: The last sentence is contradictory. We are recommending one word.

**Review by mportman12 — CHANGES_REQUESTED** [review]
Approved a little too soon. Left comment.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc (hunk 1 above)
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc]: Thanks, @mportman12  -- I had actually spotted that error myself but didn't get a chance to add a comment. Fixed it now anyway -- ready for review again :)

**Review by bburt-rh — APPROVED** [review]

**Comment by bburt-rh** [comment]
lgtm

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by dfitzmau — APPROVED** [review]

**Pull request merged**